pkg/kube/util.go

```diff
@@ -231,6 +231,9 @@ func GetDeployMetaFromPod(pod *kubeApiCore.Pod) (metav1.ObjectMeta, metav1.TypeM
 				name := strings.TrimSuffix(controllerRef.Name, "-"+pod.Labels["pod-template-hash"])
 				deployMeta.Name = name
 				typeMetadata.Kind = "Deployment"
+			} else if typeMetadata.Kind == "ReplicationController" && pod.Labels["deploymentconfig"] != "" {
+				deployMeta.Name = pod.Labels["deploymentconfig"]
```

Comment:
what is this 'deploymentconfig' label? I don't see any reference is kubernetes.

Reply:
It is an openshift resource: https://docs.openshift.com/container-platform/4.1/applications/deployments/what-deployments-are.html#deployments-and-deploymentconfigs_what-deployments-are We need to comment it well here and point to the openshift reference for this label.

Reply:
`ReplicationController` was replaced by deployment several years ago. Should be no use anymore

Reply:
For the openshift platform, pods which are controlled by `ReplicationController` can still be created by `DeploymentConfig`.